scripts/release-notes.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
```

Comment:
Do we need this script at all?

Reply:
i am using it for generating the release notes in the ci job.

Reply:
What I mean is: why having release notes for something we update and release continuously?

Reply:
well with this CI in place the docs will (now) be updated at the same time as the monthly release.